projects/kit/components/tiles/tiles.component.ts

```diff
@@ -65,7 +65,12 @@ export class TuiTilesComponent {
     }
 
     private filter(element?: Element): element is Element {
-        return !!this.element && !!element && this.element !== element;
```

Comment:
This is wrong, there should never be a `PointerEvent` here as an argument. The issue was cause by moving `@HostBinding` to `host` above, it used to not have any arguments and now it passes `$event` when it shouldn't. Please remove it from line 33

Reply:
I see, it uses `onEnter()` from tile.
Fixed